src/v1/models/ansible.js

```diff
@@ -51,7 +51,8 @@ export default class AnsibleModel extends KubeModel {
     const [ansibleCredentials] = await Promise.all([
       this.kubeConnector.getResources((ns) => `/api/v1/namespaces/${ns}/secrets?labelSelector=cluster.open-cluster-management.io/type=ans`),
     ]);
-    return ansibleCredentials.map((ans) => ({
+    const creds = ansibleCredentials.filter((ans) => ans.metadata.labels['cluster.open-cluster-management.io/copiedFromSecretName'] === undefined);
```

Comment:
Correct me if I'm wrong, here should be `!==` instead of `===` ? @dhaiducek

Reply:
I think this PR is looking to only filter for secrets that aren't copied (i.e. the label would be `undefined` as the code is written)

Here's the issue showing how those labels are implemented: https://github.com/open-cluster-management/backlog/issues/9108

Reply:
Though, as @gparvin mentioned it would be good to have some basic tests for this, but I understand that may not be feasible to complete by the end of today.

Reply:
I've added a test, can you review it again when you can?

Reply:
LGTM! Thanks, Gus!